src/main/java/org/folio/innreach/util/KafkaUtil.java

```diff
@@ -0,0 +1,114 @@
+package org.folio.innreach.util;
+
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j2;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.folio.innreach.batch.contribution.listener.ContributionExceptionListener;
+import org.folio.innreach.domain.dto.folio.inventorystorage.InstanceIterationEvent;
+import org.folio.innreach.external.exception.ServiceSuspendedException;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+import org.springframework.kafka.listener.ConcurrentMessageListenerContainer;
+import org.springframework.kafka.listener.ContainerProperties;
+import org.springframework.kafka.listener.DefaultErrorHandler;
+import org.springframework.util.backoff.BackOff;
+import org.springframework.util.backoff.FixedBackOff;
+
+import java.net.SocketTimeoutException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.apache.kafka.clients.consumer.ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG;
+
+@Log4j2
+@Setter
+@RequiredArgsConstructor
+public class KafkaUtil {
```

Comment:
I know this name is taken from the sample code, but it doesn't really work well for our purposes. How about something like `InitialContributionJobConsumerContainer`? It's got stuff in it that is specific to the init contribution job like the error handlers so it probably won't be reused any other place.

So remove from util directory obviously and put in a better place.